### ISSUE TITLE
Integration with iPhone contacts so you can just pick a friend and send coins

### ISSUE THREAD

**Issue by weilu** [hivewallet/hive-ios#28]
@jsuder how similar/different are osx address book and ios address book integration?

**Comment by mackuba** [comment]
I haven't done any so I don't know... I assume they might be similar. You probably make a call to the SDK asking for contacts, perhaps with some filter, user gives access to contacts or not (only first time), and you either get a list of records or you don't.

**Comment by weilu** [comment]
Do you think down the road it's worth integrating with the address book for osx and making a reusable module/library that hive-ios can share as well?

**Comment by mackuba** [comment]
I haven't really thought about how to handle this in Hive. I'm worried that it might be a Pandora's box that will open a lot of new problems about how to map addresses to contacts, how to distribute and store them etc. (see all those unresolved threads in https://github.com/hivewallet/discussions...).

**Comment by weilu** [comment]
Fair enough. Let's table this until we make more progress on the relevant discussions
